{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/careers/templates/careers/career_detail.html

```diff
@@ -1,5 +1,52 @@
 {% extends "base.html" %}
 
+{% block schema %}
+  <script type="application/ld+json">
+    {
+      "@context": "http://schema.org",
+      "@type": "JobPosting",
+      {% if career.estimated_salary %}"estimatedSalary": "{{ career.estimated_salary }}",{% endif %}
```

Comment:
It is probably better to move all of this template logic to the model/instance and JSON-encode it properly.